includes/class-wc-payments-features.php

```diff
@@ -104,7 +104,9 @@ public static function is_platform_checkout_eligible() {
 	 * @return bool
 	 */
 	public static function is_documents_section_enabled() {
-		return '1' === get_option( '_wcpay_feature_documents', '0' );
+		$account              = WC_Payments::get_database_cache()->get( WCPay\Database_Cache::ACCOUNT_KEY );
+		$is_documents_enabled = is_array( $account ) && ( $account['is_documents_enabled'] ?? false );
+		return '1' === get_option( '_wcpay_feature_documents', $is_documents_enabled ? '1' : '0' );
```

Comment:
Good catch on this change here. Having the ability to disable or enable the feature through the flag allows for easier testing.